src/Plugin.php

```diff
@@ -180,6 +180,7 @@ public static function bjll_skip_classes($skip_classes) {
    */
   public static function renderTemplate(array $template_subpathnames, array $variables = []) {
     $template_pathname = locate_template($template_subpathnames, FALSE, FALSE);
+    $group_id = uniqid('gallerya-', true); // Used to "group" images when open.
```

Comment:
I'd rather add this code to the function which is calls the render template function here: https://github.com/netzstrategen/wordpress-gallerya/blob/master/src/Plugin.php#L138-L141. This aligns more with SRP (Single-Responsibility Principle) where each function should have a single purpose. It's more tricky to figure out where the `group_id` variable is actually coming from as the 2nd parameter is responsible for passing the variables into the template (but now has the side effect of passing another variable).

Furthermore booleans should always be uppercase :-)
```suggestion
    $group_id = uniqid('gallerya-', TRUE); // Used to "group" images when open.
```